pages/common/grep.md

```diff
@@ -3,34 +3,38 @@
 > Matches patterns in input text.
 > Supports simple patterns and regular expressions.
 
+- Search for a pattern:
+
+`grep {{^regex$}} {{path/to/file}}`
+
 - Search for an exact string:
 
-`grep {{search_string}} {{path/to/file}}`
+`grep -F {{search_string}} {{path/to/file}}`
 
 - Search in case-insensitive mode:
 
-`grep -i {{search_string}} {{path/to/file}}`
+`grep -i {{^regex$}} {{path/to/file}}`
 
 - Search recursively (ignoring non-text files) in current directory for an exact string:
 
-`grep -RI {{search_string}} .`
+`grep -RIF {{search_string}} .`
 
 - Use extended regular expressions (supporting `?`, `+`, `{}`, `()` and `|`):
 
 `grep -E {{^regex$}} {{path/to/file}}`
 
 - Print 3 lines of [C]ontext around, [B]efore, or [A]fter each match:
```

Comment:
My suggestion is to remove this one. The example after that already shows filename and line numbers, which is enough context to find the match.

Reply:
I don't know. I find the lines either side as context much more useful myself.

Reply:
I agree. I almost always grep with a context. Would suggest keeping this one.